packages/@aws-cdk/region-settings/lib/settings/aws.ts

```diff
@@ -0,0 +1,97 @@
+import { Region } from '../region';
+
+// tslint:disable:variable-name Just replacing - with _ in region names...
+
+export const us_east_1 = new Region('us-east-1', {
+  apiGatewayRegionalHostedZoneId: 'Z1UJRXOUMOOFQ8',
+  s3StaticWebsiteLegacyNameFormat: true,
```

Comment:
Wow, I had to go hunting for these.

I think the only way to model this information in an extensible and manageable way is going to be using a rules engine.